data/policies/implicit_this.cas

```diff
@@ -0,0 +1,10 @@
+resource foo {
+	fn read(domain source) {
+		allow(source, this, file, read);
+	}
+}
+
+domain bar {
+	//equivalent to foo.read(this)
+	foo.read();
```

Comment:
I the `foo.read(this)` syntax is tested somewhere else, right?

Reply:
Yep, lots of places.  Alphabetically first is here: https://github.com/dburgener/cascade/blob/33ddf1a20e517fcdbfc18261d86360538acc5485/data/policies/alias.cas#L14